server/resources/resources.test.js

```diff
@@ -385,4 +385,33 @@ describe("/api/resources", () => {
 			expect(published).toMatchObject({ publisher: admin.id, source: user.id });
 		});
 	});
+
+	describe("POST / - Auto publish for the admin resource", () => {
```

Comment:
💡 This should really appear under the existing `POST /` group of tests; you can _nest_ `describe` blocks where you really need separate contexts:

```javascript
describe("/api/resources", () => {
  describe("POST /", () => {
    describe("as a user", () => {
      // ...
    });
  
    describe("as an admin", () => {
      // ...
    });
  });
});
```